### DISCUSSION TITLE
Steps to Unlock Viewing Cage

### DISCUSSION THREAD

**Discussion by KrisCris** [KrisCris/Palworld-Pal-Editor#24] [Q&A] [answered]
After selecting a player, there is a blue cube icon showing up right next to the `PLAYER LIST` text.

![image](https://github.com/KrisCris/Palworld-Pal-Editor/assets/38860226/284caccf-7bb7-499e-9def-b7e4a1d464a1)

**Comment by KrisCris** [comment, marked as answer]
👆

**Comment by mizifih** [comment]
@KrisCris, would be possible to make the Pals in the viewing cage invisible with your tool? That would be super incredible. Here's a mod that does it: https://www.nexusmods.com/palworld/mods/1149. Maybe it's doable.

**Reply by KrisCris** [reply]
Not doable

**Comment by jvelazcodev** [comment]
@KrisCris hey im hosting a server on G-Portal but i can use FTP to transfer my save files back and forth. I was able edit the save just like the screenshots and was able to save without problems but once I transfer the save back over to the server and log in it does not show up on the build screen. Any suggestions?
